examples/node-app-mssql-ts/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM node:14-slim
+FROM node:16-slim
```

Node 16 is going away soon. Let's use Node 18.

```suggestion
FROM node:18-slim
```